app/delivery/send_to_providers.py

```diff
@@ -255,11 +253,10 @@ def get_logo_url(
 
 def get_html_email_options(
     notification,
-    provider,
 ):
```

Comment:
nit: Can we make this function definition one line, especially since there's only one param now?
It would be nice to have the return type of `dict` added while we are at it.

Reply:
Good eye. Updated.